nomination/templates/nomination/url_listing.html

```diff
@@ -264,6 +275,13 @@ <h4 class="panel-title">
                 source: {{ institutions|safe }}
             });
             initForm();
+            {% if metadata_vals %}
+                {% for met,vals in metadata_vals %}
+                    {% if met.form_type == "select" or met.form_type == "selectsingle" %}
+                        new SlimSelect({select: '#{{ met.metadata.name }}'});
+                    {% endif %}
+                {% endfor %}
+            {% endif %}
         });
```

Comment:
To handle being able to deselect on a "selectsingle" we may want to include `allowDeselect: true` in the `SlimSelect` call in url_add.html and url_listing.html.

Reply:
okie dokie